addons/point_of_sale/static/src/js/Popups/AbstractAwaitablePopup.js

```diff
@@ -55,4 +55,9 @@ export class AbstractAwaitablePopup extends Component {
     async getPayload() {
         return null;
     }
+    onKeyup(e) {
+        if ((e.key === "Enter" || e.keyCode === 13) && e.target.nodeName === "INPUT") {
+            this.confirm();
+        }
+    }
```

Comment:
Based on `_onWindowKeyup`, it seems that any popup can be configured to have `confirmKey` and/or `cancelKey` props. Maybe instead of introducing this new method, we should utilize the props.

Perhaps, to solve this problem, we introduce a `defaultProps = {'confirmKey': 'Enter' }` to the Popups we want this behaviour to have.